src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -542,17 +536,18 @@
                 IsHitTestVisible="False"
                 SnapsToDevicePixels="True" />
 
-        <!--  Displays when the node is frozen  -->
-        <Border Name="nodeIsFrozenOverlay"
+        <!--  Displays when the node is Frozen/Warning/Error state -->
+        <Border Name="nodeColorOverlay"
                 Grid.Row="1"
                 Grid.RowSpan="4"
                 Grid.ColumnSpan="3"
                 Margin="-8"
-                Background="{StaticResource NodeFrozenOverlayBackground}"
-                Canvas.ZIndex="0"
+                Background="{Binding NodeOverlayColor, UpdateSourceTrigger=PropertyChanged}"
+                Canvas.ZIndex="7"
+                Opacity="0.5"
                 CornerRadius="8,8,0,0"
-                IsHitTestVisible="True"
-                Visibility="{Binding Path=IsFrozen, Converter={StaticResource BooleanToVisibilityCollapsedConverter}, UpdateSourceTrigger=PropertyChanged}" />
+                IsHitTestVisible="False"
```

Comment:
hmm - so does this change mean you can't drag nodes that have the overlay visible?

Reply:
The node itself still has a hit test - just not the overlay. Dragging should be fine